src/users/users.module.ts

```diff
@@ -1,12 +1,19 @@
-import { Module } from '@nestjs/common';
+import { CacheModule, Module } from '@nestjs/common';
 import { UsersService } from './users.service';
 import { UsersResolver } from './users.resolver';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { User } from './entities/user.entity';
-
+import * as redisStore from 'cache-manager-redis-store';
 @Module({
-  imports: [TypeOrmModule.forFeature([User])],
+  imports: [
+    CacheModule.register({
+      store: redisStore,
+      host: 'localhost',
+      port: 6379,
+    }),
```

Comment:
Do you use caching in user module?
If yes, seems like duplication of providing a store, consider using a separate custom module where you can once register a cache module with storage and use it in future.

Reply:
I do not figure out how to use it only in one place.
It seems like, it needs connecting CacheModule everywhere where it will be using, so It connected in auth - where I set data, and in user because It reads data in user.guard from user.resolver